src/edges_analysis/analysis/filters.py

```diff
@@ -909,19 +909,20 @@ def fnc(
         mask = (data.raw_frequencies >= freq_range[0]) & (
             data.raw_frequencies <= freq_range[1]
         )
+        if data.spectrum.ndim == 2:
```

Comment:
don't need this check.

Reply:
@steven-murray this is ready to be merged?